<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="107.3630" tests="13" failures="2" skipped="1">
  <testsuite name="Root Suite" timestamp="2022-10-15T12:55:46" tests="0" file="cypress/e2e/process.cy.ts" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="Process" timestamp="2022-10-15T12:55:46" tests="2" time="29.1440" failures="0">
    <testcase name="Process should preserve condition on link move (switch)" time="19.1650" classname="should preserve condition on link move (switch)">
    </testcase>
    <testcase name="Process should preserve condition on link move (filter)" time="9.9790" classname="should preserve condition on link move (filter)">
    </testcase>
  </testsuite>
  <testsuite name="initially clean" timestamp="2022-10-15T12:56:17" tests="4" time="21.6610" failures="2">
    <testcase name="Process initially clean should allow rename" time="10.6520" classname="should allow rename">
    </testcase>
    <testcase name="Process initially clean should allow archive with redirect to list" time="0.0000" classname="should allow archive with redirect to list">
      <failure message="Timed out retrying after 60000ms: Expected to find content: &apos;/^archived visible$/i&apos; but never did." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 60000ms: Expected to find content: '/^archived visible$/i' but never did.
    at Context.eval (cypress/e2e/process.cy.ts:43:59)]]></failure>
    </testcase>
    <testcase name="Process initially clean should open properites from tips panel" time="0.0000" classname="should open properites from tips panel">
      <failure message="Timed out retrying after 10050ms: `cy.click()` failed because this element is detached from the DOM.

`&lt;a class=&quot;node-error-link&quot; href=&quot;/&quot;&gt;properties&lt;/a&gt;`

Cypress requires elements be attached in the DOM to interact with them.

The previous command that ran was:

  &gt; `cy.should()`

This DOM element likely became detached somewhere between the previous and current command.

Common situations why this happens:
  - Your JS framework re-rendered asynchronously
  - Your app code reacted to an event firing and removed the element

You typically need to re-query for the element or add &apos;guards&apos; which delay Cypress from running new commands.

https://on.cypress.io/element-has-detached-from-dom" type="CypressError"><![CDATA[CypressError: Timed out retrying after 10050ms: `cy.click()` failed because this element is detached from the DOM.

`<a class="node-error-link" href="/">properties</a>`

Cypress requires elements be attached in the DOM to interact with them.

The previous command that ran was:

  > `cy.should()`

This DOM element likely became detached somewhere between the previous and current command.

Common situations why this happens:
  - Your JS framework re-rendered asynchronously
  - Your app code reacted to an event firing and removed the element

You typically need to re-query for the element or add 'guards' which delay Cypress from running new commands.

https://on.cypress.io/element-has-detached-from-dom
    at $Cy.ensureAttached (http://host.docker.internal:3000/__cypress/runner/cypress_runner.js:147481:76)
    at runAllChecks (http://host.docker.internal:3000/__cypress/runner/cypress_runner.js:133346:12)
    at retryActionability (http://host.docker.internal:3000/__cypress/runner/cypress_runner.js:133426:16)
    at tryCatcher (http://host.docker.internal:3000/__cypress/runner/cypress_runner.js:11318:23)
    at Promise.attempt.Promise.try (http://host.docker.internal:3000/__cypress/runner/cypress_runner.js:8592:29)
    at whenStable (http://host.docker.internal:3000/__cypress/runner/cypress_runner.js:152675:65)
    at <unknown> (http://host.docker.internal:3000/__cypress/runner/cypress_runner.js:152105:14)
    at tryCatcher (http://host.docker.internal:3000/__cypress/runner/cypress_runner.js:11318:23)
    at Promise._settlePromiseFromHandler (http://host.docker.internal:3000/__cypress/runner/cypress_runner.js:9253:31)
    at Promise._settlePromise (http://host.docker.internal:3000/__cypress/runner/cypress_runner.js:9310:18)
    at Promise._settlePromise0 (http://host.docker.internal:3000/__cypress/runner/cypress_runner.js:9355:10)
    at Promise._settlePromises (http://host.docker.internal:3000/__cypress/runner/cypress_runner.js:9435:18)
    at Promise._fulfill (http://host.docker.internal:3000/__cypress/runner/cypress_runner.js:9379:18)
    at <unknown> (http://host.docker.internal:3000/__cypress/runner/cypress_runner.js:10993:46)
From Your Spec Code:
    at Context.eval (cypress/e2e/process.cy.ts:58:85)]]></failure>
    </testcase>
    <testcase name="Process initially clean should import JSON and save" time="11.0090" classname="should import JSON and save">
    </testcase>
  </testsuite>
  <testsuite name="with data" timestamp="2022-10-15T12:58:08" tests="7" time="56.5580" failures="0">
    <testcase name="Process with data should allow drag node" time="7.0550" classname="should allow drag node">
    </testcase>
    <testcase name="Process with data should allow drag component and drop on edge" time="11.3670" classname="should allow drag component and drop on edge">
    </testcase>
    <testcase name="Process with data should have counts button and modal" time="6.4290" classname="should have counts button and modal">
    </testcase>
    <testcase name="Process with data should return 400 status code and show info about required comment" time="6.5630" classname="should return 400 status code and show info about required comment">
    </testcase>
    <testcase name="Process with data should not have &quot;latest deploy&quot; button by default" time="13.4250" classname="should not have &quot;latest deploy&quot; button by default">
    </testcase>
    <testcase name="Process with data should display some node details in modal" time="11.7190" classname="should display some node details in modal">
    </testcase>
  </testsuite>
</testsuites>